specs/DesignDiscussions/Localization.md

```diff
@@ -0,0 +1,109 @@
+# Localization in Adaptive Cards
+
+Currently Adaptive Cards has minimal localization support. This document covers areas where localization may be necessary. Currently this document does not cover detailed solutions to these scenarios, though those solutions will be added here as work progresses.
```

Comment:
Given the ever-expanding scope of "Adaptive Cards" do we need to clarify the scope of the discussion here? For eg it doesn't cover the broader AdaptiveCards ecosystem at this point (adapticecards.io website, designer, CMS, Templating SDKs etc) but focused on our renderer SDKs? Maybe like an explicit call out of hte "Goals/Non-Goals" and a clear articulation of  "Out of scope" aspects? Alternatively/Eventually we could have a parallel doc that might talk more holistically about localizability + localization of AC including our ecosystem and offerings beyond the renderers ?

Reply:
This document does reference all the aspects of the ecosystem you call out as you can see below. Admittedly there may be more detail on the renderer issues in this early draft, but I envision the "Documentation and Tooling" and "Components/Templating" sections below to be as relevant as the other sections, and as likely to be drilled into in the future as we prioritize this work. The goal of this early draft is breadth rather than depth, to call out all areas where localization may be needed. We can then decide if and where we want to expend resources to dig in further. 

---
In reply to: [509283708](https://github.com/microsoft/AdaptiveCards/pull/4939#discussion_r509283708) [](ancestors = 509283708)